### ISSUE TITLE
Alarm-notification sound not looping in iOS

### ISSUE THREAD

**Issue by hardikjain29** [emekalites/react-native-alarm-notification#83]
First of all, thank you so much for this package!

I have an issue on iOS, even when I pass `loop_sound: true` the notification sound doesn't keep ringing.

Do I have to pass anything else to it?

**Comment by emekalites** [comment]
Hi, @hardikjain29  the iOS is till under development.

**Comment by hardikjain29** [comment]
Ohh okay! Waiting eagerly for it. Wish I could help here! 😕

Thanks a lot!

**Comment by emekalites** [comment]
I have updated the plugin to v1.7.0.
I struggled a lot with the sound for iOS. firstly iOS doesn't allow you to control the  notification sound so adding making it loop is difficult. secondly, I realized i cant use a custom sound longer than 30seconds because the sound ends as soon the notification banner slides out.
If there was a way to create a custom push notification banner that stays at the top until you dismiss it, i think that will enable the sound to last longer.

Try out the alarm app on your iPhone and you'll see that the notification banner doesnt close thereby allowing your phone to keep beeping until you dismiss the notification.

**Comment by hardikjain29** [comment]
Ohh, that's a bummer! Very sad that Apple doesn't allow this.

I had a thought as you said if the notification banner can stay longer, the sound could last longer, for that what I did was go to the app settings > Notification > Banner > Persistent. This will not make the banner(notification) go away until the user slides it up. But that didn't make it sound longer. (`loop_sound: true`)

I tried doing something on the iOS code side but no luck. After that I read some places and found out that Apple might be using an internal private API to make the alarm notification sound longer. They apparently don't want to expose that.

Altogether, thanks a lot for trying this out, I'll also try contributing if something strikes me and I am able to make it work.
